python/ray/experimental/dag/__init__.py

```diff
@@ -1,6 +1,13 @@
 from ray.experimental.dag.dag_node import DAGNode
 from ray.experimental.dag.function_node import FunctionNode
 from ray.experimental.dag.class_node import ClassNode, ClassMethodNode
+from ray.experimental.dag.constants import DAG_ENTRY_POINT
```

Comment:
```suggestion
from ray.experimental.dag.constants import INPUT
```

How about shortening this? I think dag.INPUT (or input_args, etc.) is pretty clear. Entrypoint sounds a bit esoteric.

Reply:
sure i think this is simpler too